gengo/Cargo.toml

```diff
@@ -15,7 +15,7 @@ max-performance = ["gix/max-performance"]
 max-performance-safe = ["gix/max-performance-safe"]
 
 [dependencies]
-gix = { version = "0.52", default-features = false }
+gix = { version = "0.53.0", default-features = false, features = ["index", "attributes", "revision"] }
```

Comment:
Nitpick: at least for the library, I'd rather have a looser version constraint like `0.53`.